pages/iam/how-to/set-up-identity-federation.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: How to set up identity federation
+description: This page shows you how to set up identity federation on your Organization
+dates:
+  validation: 2025-09-21
+  posted: 2025-08-21
+---
+
+You can set up Identity Federation at Scaleway to ensure your [members can log in via Single Sign-On (SSO)]().
+
+At Scaleway we use Security Assertion Markup Language (SAML) to provide Identity Federation. You can link user identities across multiple independent systems and organizations to enable SSO across domains. You can manage your Scaleway identities via your Identity Provider of choice, as long as the provider supports SAML.
+
+<Requirements />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
```

Comment:
We can add in the Requirements: An Identity Provider configured in your company, with all users who need to access to Scaleway already set-up in it. Typical examples of Identity providers include (not exhaustive)
- Okta
- OneLogin
- Microsoft Entra ID (prev. Azure AD)
- PingIdentity
- Google Workspace
- Keycloak